examples/PET_simulation/ScatterEstimation.par

```diff
@@ -0,0 +1,95 @@
+Scatter Estimation Parameters :=
+
+;Run in debug mode
+;; A new folder called extras will be created, in which many 
+;; extra files will be stored 
+run in debug mode := 1
+
+;; 
+;; initialisation 
+;; 
+
+; If name is provided and recomputation == true
+; the file will be saved in the disk 
+
+recompute initial activity image := 1
+initial activity image filename := 
+
+; Smoothing for activity and attenuation images 
+zoom xy := 
+zoom z := 
+
+; Measured data
+input file := 
+
+; Multuplicative data 
+
+; Attenuation data
+recompute attenuation projdata := 
+attenuation projdata filename := 
+attenuation image filename := 
+
+; Normalisation coefficients
+normalisation coefficients filename :=
+
+; Mask 
+; If !recompute mask projdata then the filename must be set.
+recompute mask projdata := 
+mask projdata filename := 
+
+; Input or output filename - depends on recompute  
+;; If recompute is set, fwd will be performed automatically
+recompute mask image := 
+mask image filename := 
+
+mask max threshold := 
+mask add scaler := 
+mask min threshold := 
+mask times scalar :=
+mask prostfilter filename := 
```

Comment:
misspelling `postfilter`. I guess I'd put this variable before the threshold etc as it's applied first (I know order of keywords doesn't matter, but easier for the user)